subsys/bluetooth/controller/hci_driver.c

```diff
@@ -533,6 +534,13 @@ static int configure_supported_features(void)
 		}
 	}
 
+#ifdef RADIO_TXP_DEFAULT
+	err = sdc_default_tx_power_set(RADIO_TXP_DEFAULT);
```

Comment:
Discussed offline. We should also add the FEM gain here so that the application can set its output power to a high value in a simple way. That will be addressed in a separate PR